src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -193,9 +195,39 @@ export class AccountDialog extends Modal {
 				for (const addedProvider of addedProviders) {
 					this.addProvider(addedProvider);
 				}
+			})
+			.then(async () => {
+				await this.addGitHubCopilotProvider();
 			});
 	}
 
+	private async addGitHubCopilotProvider(): Promise<void> {
+		const providers = this._authenticationService.getProviderIds();
+		for (const providerId of providers) {
+			if (providerId === 'github') {
+				const copilotAccounts: azdata.Account[] = [];
+
+				const allSessions = await this._authenticationService.getSessions(providerId);
+				for (const session of allSessions) {
```

Comment:
nit - using map here would be clearer

`const copilotAccounts = allSessions.map(s => ...)`